### PR TITLE
Reset mode after click

### PR DESCRIPTION
**Pull request by sfentress**
@ekosmin Can you take a look at this one?

### CODE REVIEW

**Comment by sfentress** [comment]
https://www.pivotaltracker.com/story/show/155570552

File: src/stores/RootStore.tsx [hunk 1]
```diff
@@ -51,6 +48,21 @@ const RootStore = types
     }
   }))
   .actions(self => ({
+    setActiveAssay(assayOrganelle: IOrganelleRef) {
+      self.activeAssay = assayOrganelle;
+
+      // add to locked assays immediately
+      let repeatAssay = self.lockedAssays.some((assay: IOrganelleRef) => {
```
Comment (ekosmin) [inline src/stores/RootStore.tsx]: Didn't know about `some`, nice change!

File: src/stores/RootStore.tsx [hunk 2]
```diff
@@ -51,6 +48,21 @@ const RootStore = types
     }
   }))
   .actions(self => ({
+    setActiveAssay(assayOrganelle: IOrganelleRef) {
+      self.activeAssay = assayOrganelle;
```
Comment (ekosmin) [inline src/stores/RootStore.tsx]: Hmm, is the idea of an `activeAssay` now obsolete? I think you might be able to get rid of it entirely.

**Comment by sfentress** [comment]
You're right about active assay. For the most part it's actually been removed in commits after this branch, so I won't confuse those branches by removing it here.

**Pull request merged**